utils/constants/leadslist-constant.js

```diff
@@ -0,0 +1,101 @@
+export const LEADS_DATA = [
+  {
+    id: 1,
+    title: 'Social Media Marketing',
+    date: 'Tue 21 Nov, 2023 11:34 AM',
+    assignedByName: 'Saidul M Khan',
+    assignedByNumber: '+88-01623364856',
+    assignedToName: 'Rezaul Mursalin Khan',
+    status: 'Hot',
```

Comment:
Keep status into `common-constant.ts` file, we can keep it as: 
```javascript 
LEAD_STATUS = {
  Hot: 'Hot',
  InProgress: 'In Progress',
......................................
.....................................
......................................
};
```
Then we can use it here as:
```javascript
status: LEAD_STATUS.Hot,
```